docs/features/column-level-lineage.md

```diff
@@ -34,6 +38,58 @@ The transformation type is also displayed for each column, which will help you u
 | Unknown | We have no information about the transformation type. This could be due to a parse error, or other unknown reason. | 
 
 
-## Limitation
+## Impact Radius of a Column
+
+The **right side of the Column-Level Lineage (CLL)** graph represents the **impact radius** of a selected column.
+This view helps you quickly understand what will be affected if that column changes.
+
+### What does the impact radius include?
+
+- **Downstream columns** that directly reference the selected column
+- **Downstream models** that directly depend on the selected column
+- **All indirect downstream columns and models** that transitively depend on it
+
+This helps you evaluate both the direct and downstream effects of a column change, making it easier to understand its overall impact.
+
+
+### Example: Simplified Model Chain
+![alt text](../assets/images/features/cll-2.png){: .shadow}
+
```

Comment:
Some textual explanation of the image would help, in addition to the raw SQL from the models.
To help the reader understand how CLL can help them to visualize the relations. Of course, they can figure it out from the SQL by themself, but we can speed that up with a brief, plain language, description

Reply:
I have updated the example segment. PTAL

![image](https://github.com/user-attachments/assets/08bbe66f-332d-4d35-98e9-cfcfa6ec6c7d)